src/auth/index.js

```diff
@@ -208,19 +213,28 @@ export default class Auth {
     return this.storage.get(authCommon.REFRESH_TOKEN_KEY);
   }
 
-  set(json) {
+  setAuthType(authType) {
+    return new Promise((resolve) =>
+      resolve(this.storage.set(authCommon.USER_LOGGED_IN_PT_KEY, authType))
+    );
+  }
+
+  set(json, authType) {
     if (!json) {
       return;
     }
 
     let newUserAuth = {};
     return new Promise((resolve) => {
+      if (!authType) { resolve(); }
+      else { resolve(this.storage.set(authCommon.USER_LOGGED_IN_PT_KEY, authType)); }
```

Comment:
I think you could just use `return` statements here and not need to involve `resolve` altogether.

Reply:
done